doc/man3/SSL_CTX_set_cookie_generate_cb.pod

```diff
@@ -0,0 +1,60 @@
+=pod
+
+=head1 NAME
+
+SSL_CTX_set_cookie_generate_cb,
+SSL_CTX_set_cookie_verify_cb
+- Callback functions for DTLS and statless TLS1.3 cookies
+
+=head1 SYNOPSIS
+
+ #include <openssl/ssl.h>
+
+ void SSL_CTX_set_cookie_generate_cb(SSL_CTX *ctx,
+                                     int (*app_gen_cookie_cb) (SSL *ssl,
+                                                               unsigned char
+                                                               *cookie,
+                                                               unsigned int
+                                                               *cookie_len));
+ void SSL_CTX_set_cookie_verify_cb(SSL_CTX *ctx,
+                                   int (*app_verify_cookie_cb) (SSL *ssl,
+                                                                const unsigned char
+                                                                *cookie,
+                                                                unsigned int
+                                                                cookie_len));
+
+=head1 DESCRIPTION
+
+SSL_CTX_set_cookie_generate_cb() sets the callback used by L<DTLSv1_listen(3)>
+and L<SSL_stateless(3)> to generate the application-controlled portion of the
+cookie provided to clients in the HelloRetryRequest transmitted as a response to
+a ClientHello with a missing or invalid cookie. app_gen_cookie_cb() must write
```

Comment:
The above describes SSL_stateless() which is a TLSv1.3 thing, but not so much DTLSv1_listen(). DLTSv1_listen() (which only supports <= DTLSv1.2), the callback provides the entire cookie (i.e. not just "the application-controlled portion"). Also it is created in response to a HelloVerifyRequest message in DTLSv1.2 and below.